object_detector_cdt/include/object_detector_cdt/object_detector.h

```diff
@@ -24,12 +24,13 @@
 #include <opencv2/highgui.hpp>
 #include <iostream>
 
-#define MIN_BBOX_WIDTH 50
-#define MIN_BBOX_HEIGHT 50
-#define MIN_OBJECT_DEPTH 1
+#define MIN_BBOX_WIDTH 75
```

Comment:
These would probably be better in a launch file but don't bother changing it now.